debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MetadataIT.java

```diff
@@ -77,7 +77,8 @@ public void shouldLoadMetadataViaJdbc() throws SQLException {
             assertThat(person.columnWithName("age").typeName()).isEqualTo("INT");
             assertThat(person.columnWithName("age").jdbcType()).isEqualTo(Types.INTEGER);
             assertThat(person.columnWithName("age").length()).isEqualTo(10);
-            assertThat(person.columnWithName("age").scale().get()).isEqualTo(0);
+            assertThat(!person.columnWithName("age").scale().isPresent()
```

Comment:
@vamossagar12 Why is it that it's testing for two cases here? I.e. under which circumstances would which part of the OR hold true? Given that `age` is an INTEGER column, I'd expect that the scale never is present for it?

Reply:
that's something on @jpechane 's suggestion. He said we can keep the code in such a way so that we can switch across drivers' versions seamlessly.

Reply:
Ok, thanks for the clarification. After discussing further with Jiri, we agreed that we'll do the transition to driver 8.x and not "support" usage of the older driver. After all, it's an implementation detail of the connector which shouldn't be customizeable by the user. Note that the driver docs highly recommend the update:

> MySQL Connector/J 8.0 is highly recommended for use with MySQL Server 8.0, 5.7, 5.6, and 5.5. Please upgrade to MySQL Connector/J 8.0.

In that light I'll adjust this assertion to only pass with the new behavior and add a comment about that fact and then merge the PR.